keymanapp/keyman
# chore(common): Check in crowdin strings for Simplified Chinese

This pulls in the crowdin strings for Simplified Chinese

TODO:
- [x] Run xcode to bring in strings for iOS project

## User Testing
Setup - load the PR build

* **TEST_ZH** - Test Keyman apps display Chinese UI
1. Load Keyman for Android
2. From the "Settings" menu --> Display Language --> select "(Simplified Chinese)"
4. Go back to "Settings" menu
5. Verify most of the menus are now in simplified Chinese (Some of the newer 15.0 strings aren't on crowdin yet)
6. Go back to "Settings" --> Display Language --> English
7. Go back to "Settings" menu
8. Verify the menus are back to English
